Dockerfile

```diff
@@ -107,8 +107,8 @@ COPY --from=builder /usr/src/app /usr/src/app
 
 COPY . /usr/src/app
 
-RUN mkdir -p /data && chown node:node /data
-VOLUME /data
+RUN mkdir -p /data /usr/src/app/public/files && chown node:node /data /usr/src/app/public/files
```

Comment:
the only thing I don't really like is making this folder with the dockerfile . since tileserver-gl can run directly without docker, it would need that folder there in that use case also. So maybe we should either make the directory with a default file inside, of maybe find a way to create it with the package.json npm prepare..

Reply:
I added the folder to the repository (and excluded .gitignore files for docker build).

Reply:
sounds goo to me. this can be removed now right?

Reply:
I meant can we remove the changes in this file. Since the folder exists we don't need to create it here anymore.

Reply:
Sorry, I was a bit confused.

In fact the `mkdir` for public/files may be removed, but due to the '-p' it does not hurt and ensures, that the path, which will be chown'ed and exposed as volume is really there. 
If I remove/change it, I would revert the whole file, because the volume depends on the configuration and everything will still  work with the default configuration, because the mount may be applied without a exposed volume of an empty directory. (This may be different, if there was a default config file included in the image.)

In short: It's up to you ;-)

Reply:
I would rather keep it as it was before since it shouldn't really be needed no.

Reply:
Done